doc_template/visual_behavior_ophys_nwb.rst

```diff
@@ -0,0 +1,110 @@
+
+.. figure:: /_static/behavior_ophys_nwb_fig-3.png
+
+   **Figure 3. Location of dF/F data**
+
+
+HDFView provides a very efficient way to browse the contents of an NWB file.
+
+Below is a brief description and location for several types of data in the NWB file.
+
++-----------+-----------------------+------------------------------------------------------------------------------+----------------------------------------------+
+| Category  | Data                  | Location                                                                     | SDK function(s)                              |
++===========+=======================+==============================================================================+==============================================+
+| Metadata  | Cre line              | /general/subject/genotype                                                    |                                              |
++           +-----------------------+------------------------------------------------------------------------------+                                              |
+|           | Imaging depth         | /general/metadata                                                            |                                              |
++           +-----------------------+------------------------------------------------------------------------------+                                              |
+|           | Target structure      |                                                                              |                                              |
++           +-----------------------+------------------------------------------------------------------------------+                                              |
+|           | Session type          | /general/task_parameters                                                     |                                              |
++-----------+-----------------------+------------------------------------------------------------------------------+----------------------------------------------+
+| Stimulus  | Natural Images        | /stimulus/presentation/Natural_Images_Lum_Matched_set_training_2017.07.14    |                                              |
+|           |                       |                                                                              |                                              |
++           +-----------------------+------------------------------------------------------------------------------+----------------------------------------------+
+|           | Stimulus Distribution | /general/task_parameters                                                     |                                              |
+|           |                       |                                                                              |                                              |
```

Comment:
Stimulus category is missing
Stimulus Templates: "/stimulus/templates"
Presentation Intervals: "/intervals/*_presentations"

There should be a `Behavior` category that includes:
Licks
Running
Rewards
Trials (could possibly fit in another category)
Eye tracking

Missing Ophys stuff:
Average project image
Max project image